src/main/java/com/google/sps/servlets/study-resources/YouTubeServlet.java

```diff
@@ -82,42 +86,49 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       System.out.println("GET Request: YouTubeServlet");     
       try {
         YouTube youtubeService = getService();
-        long results = NUM_VIDEOS;
+        long results = 0;
+        if (NUM_VIDEOS == 0) {
```

Comment:
results = Math.Min(1, NUM_VIDEOS)

Any particular reason why you need long here instead of int?

Reply:
The YouTube API's search.list() takes in a long for the maxResults parameter.